botocore/client.py

```diff
@@ -191,6 +191,14 @@ def _inject_s3_configuration(self, config_kwargs, scoped_config,
 
         config_kwargs['s3'] = s3_configuration
 
+    def _resolve_verify_value(self, provided_verify_value, scoped_config):
+        # If a user provides a value for "verify" then we don't do anything.
+        # We always use explicit values provided.
+        if provided_verify_value is not None:
+            return provided_verify_value
+        elif scoped_config is not None:
+            return scoped_config.get('ca_bundle')
+
```

Comment:
You're correct, this doesn't affect anything because we always return immediately if the user passed in some value for `verify`.

This would also never change because we can't change the default value of the `verify` param from `None` as that would be a backwards incompatible change.